packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -109,6 +111,15 @@ class MediaQueryData {
   ///  * [MediaQuery.textScaleFactorOf], a convenience method which returns the
   ///    textScaleFactor defined for a [BuildContext].
   final double textScaleFactor;
+  
+  /// The current brightness mode of the host platform.
+  /// 
+  /// For example, starting in Android Pie, battery saver mode asks all apps to
+  /// render in a "dark mode". That change is reflected in this property.
+  /// 
+  /// Not all platforms necessarily support a concept of brightness mode. Those
+  /// platforms will report [PlatformBrightness.no_preference] in this property.
```

Comment:
Other enums in dart:ui are camelCased

Reply:
I'm in the process of trying to remove this enum and replace it with `Brightness`